examples/nfct_test.rs

```diff
@@ -0,0 +1,170 @@
+#![no_std]
+
+extern crate alloc;
+extern crate lang_items;
+extern crate libtock_drivers;
+
+use core::fmt::Write;
+use libtock_core::result::CommandError;
+use libtock_drivers::console::Console;
+use libtock_drivers::nfc::NfcTag;
+use libtock_drivers::nfc::RecvOp;
+use libtock_drivers::result::FlexUnwrap;
+use libtock_drivers::result::TockError;
+use libtock_drivers::timer;
+use libtock_drivers::timer::Timer;
+use libtock_drivers::timer::Timestamp;
+
+#[allow(dead_code)]
```

Comment:
Why are these functions potentially dead code?

Reply:
At the time I used the feature flag that hid all the nfc code, so the functions were left unused. And I'll reuse it again now as Guillaume suggested it would be better for cargo checks.